wpiutil/src/main/native/cpp/timestamp.cpp

```diff
@@ -89,6 +115,12 @@ uint64_t wpi::NowDefault() {
 static std::atomic<uint64_t (*)()> now_impl{wpi::NowDefault};
 
 void wpi::SetNowImpl(uint64_t (*func)(void)) {
+#ifdef __FRC_ROBORIO__
+  if (!global) {
+    int32_t status = 0;
+    global.reset(fpga::tGlobal::create(&status));
```

Comment:
This actually isn't going to fix the issue, because the hal tGlobal instance will still be destructed. We probably just need a different function that we call exclusively from the rio hal, that creates this tGlobal, and then creates a wpiutil local copy of what exists in HAL_GetFPGATime. That way time doesn't touch the HAL at all.